core/iwasm/common/wasm_shared_memory.c

```diff
@@ -119,7 +119,9 @@ notify_stale_threads_on_exception(WASMModuleInstanceCommon *module_inst)
                          (void *)&total_elem_count);
 
     /* allocate memory */
-    if (!(args.addr = wasm_runtime_malloc(sizeof(void *) * total_elem_count))) {
+    const uint32 total_elem_count_size = sizeof(void *) * total_elem_count;
```

Comment:
that probably should be `size_t`? Also, should we check for overflow?

Reply:
done